utests/tests/ptf/test_mask.py

```diff
@@ -53,3 +55,79 @@ def test_mask__mask_has_problem_with_conditional_fields(self):
             255,
             255,
         ], "Only gpid field should be masked"
+
+    def test_mask__conditional_field__gpid_should_be_masked_correctly(self):
+        masked_simple_vxlan = Mask(simple_vxlan_packet(vxlan_flags="G"))  # type: Mask
```

Comment:
I don't like using `simple_vxlan_packet` in these tests (the same goes with `simple_ip_packet`). They're meant to test mask mechanism. If there's a bug in this function, then most of mask tests will also fail, regardless mask state itself.
Maybe better approach is to create frames manually (using fixtures, for example)?

Reply:
Two new fixtures were created, booth for direct use with the Scapy, `scapy_simple_tcp_packet` & `scapy_simple_vxlan_packet`.

Reply:
The advantage of using simple_vxlan_packet is that it works with scapy and should work with another pluggable packet module.  If someone has their own pluggable packet module it would be nice to allow them to run a variety of tests with their module.